expression/builtin_vectorized.go

```diff
@@ -115,18 +115,24 @@ func (c *vecRowConverter) evalInt(row chunk.Row) (val int64, isNull bool, err er
 	if !c.builtinFunc.vectorized() {
 		return c.builtinFunc.evalInt(row)
 	}
-	var sel [1]int
-	sel[0] = row.Idx()
 	var buf *chunk.Column
 	if buf, err = c.builtinFunc.get(types.ETInt, 1); err != nil {
 		return
 	}
 
 	input := row.Chunk()
-	oriSel := input.Sel()
-	input.SetSel(sel[:])
-	err = c.builtinFunc.vecEvalInt(input, buf)
-	input.SetSel(oriSel)
+	if input == nil {
+		// it happens when the user ensure the leaf node is a Constant and then put an empty Row:
```

Comment:
The way to handle this case now works but not graceful.
Would you have some other ideas to solve this problem?
@zz-jason @XuHuaiyu